linera-core/src/node.rs

```diff
@@ -164,6 +164,9 @@ pub enum NodeError {
     SubscriptionError { transport: String },
     #[error("Failed to subscribe; tonic status: {status}")]
     SubscriptionFailed { status: String },
+
+    #[error("Failed to make a chain info query on the local node: {error}")]
+    LocalNodeQuery { error: String },
```

Comment:
Not needed. Also, we probably wouldn't wrap our own errors as strings.

Reply:
I think the reason that lots of fields in `NodeError` are `String` representations of other errors is that `communicate_with_quorum` uses a `HashMap` to group the errors, so it needs to implement `Hash + Eq + Clone`. Some error types from external crates don't, and neither do `LocalNodeError` or `ChainClientError`.